tests/sync/session/session.cpp

```diff
@@ -856,27 +865,28 @@ TEST_CASE("sync: client resync") {
         // test local realm that changes were persisted
         REQUIRE_THROWS(realm->refresh());
         auto table = ObjectStore::table_for_object_type(realm->read_group(), "object");
-        REQUIRE(table->get_column_count() == 2);
+        REQUIRE(table->get_column_count() == 3);
         REQUIRE(table->begin()->get<Int>("value2") == 123);
         REQUIRE_THROWS(realm->refresh());
         // test resync'd realm that changes were overwritten
         realm = Realm::get_shared_realm(config);
         table = ObjectStore::table_for_object_type(realm->read_group(), "object");
         REQUIRE(table);
-        REQUIRE(table->get_column_count() == 1);
+        REQUIRE(table->get_column_count() == 2);
         REQUIRE(!bool(table->get_column_key("value2")));
     }
 
     config.sync_config->client_resync_mode = ClientResyncMode::Recover;
 
     SECTION("add table without pk in recovered transaction") {
-        auto realm = trigger_client_reset([](auto& realm) {
+        auto realm = trigger_client_reset([&](auto& realm) {
             realm.update_schema({
                 {"object2", {
+                    {"_id", PropertyType::Int, Property::IsPrimary{true}},
```

Comment:
This is no longer testing what the test says it's testing. This section should probably just be deleted since it's no longer applicable.

Reply:
👍